deploy/kubernetes/attacher.yaml

```diff
@@ -73,6 +73,9 @@ spec:
         app: csi-attacher-s3
     spec:
       serviceAccount: csi-attacher-sa
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: "Exists"
```

Comment:
Just FYI, this does not force it to only run on master nodes, it will just allow it to be scheduled on master nodes. If there are no nodes with this taint, it will still be scheduled.

Reply:
Yes, you are right, I will fix it later

Reply:
No need to fix it, this is a good thing. If it would limit to run only on master nodes, this would actually break on some platforms as not all of them (for example GKE) allow you to schedule workloads on master nodes. So I think it should stay as is, so it can be scheduled on master nodes but does not require to.